theano/tensor/opt.py

```diff
@@ -2467,8 +2467,7 @@ def local_inplace_setsubtensor(node):
 def local_inplace_incsubtensor1(node):
     """ also work for GpuAdvancedIncSubtensor1 """
     if isinstance(node.op, AdvancedIncSubtensor1) and not node.op.inplace:
-        new_op = node.op.__class__(
-            inplace=True, set_instead_of_inc=node.op.set_instead_of_inc)
+        new_op = node.op.clone_inplace()
```

Comment:
The name make_new_inplace already exist at one place for this. I prefer the name clone_inplace. Just change the old place to reuse the new name. This wasn't documented. Add this in the op documentation.